app/decorators/users/setting_decorator.rb

```diff
@@ -0,0 +1,7 @@
+module Users
+  class SettingDecorator < ApplicationDecorator
```

Comment:
I'm not sure I'd bother with a decorator here. One advantage of dedicated settings models is that we don't have to feel so bad about adding the occasional wrapper method to the model itself. Maybe something like this directly in `Users::Setting` is good enough?

```ruby
def resolved_font_name
  config_font.gsub("default", Settings::UserExperience.default_font)
end
```